style.css

```diff
@@ -0,0 +1,143 @@
+:root {
+  --bg-purple: #6070ff;
+  --bg-dark-blue: #172b4d;
+  --bg-lavender: #ebebff;
+  --bg-slate-grey: #505f79;
+  --text-dark-blue: #344563;
+  --font-poppins: "Poppins", sans-serif;
+}
+
+*,
+*::after,
+*::before {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: white;
+}
+
+html body {
+  max-width: 100%;
+  overflow-x: hidden;
+}
```

Comment:
- [ ] The page is scrolling horizontally in the mobile version, Please 🙏 make sure you disable it. As I see in your code you have tried making overflow-x: hidden but it's not working. One reason for that could be that the size of some section or element (e.g `<h1>`, `<p>` etc) is absolute like (px, rem, etc) rather than responsive, like percentages (i.e. width: 80%). You can debug the problem using the browser developer tool 👍 

https://user-images.githubusercontent.com/44798044/219557559-2cf4ad12-a0b4-46d7-b0cf-a9aefc7bc517.mov